Play Store keeps crashing on Graphene

```
type: crash
osVersion: google/akita/akita:17/CP2A.260605.012/2026070501:user/release-keys
package: com.android.vending:85172130, targetSdk 37
process: com.android.vending:background
processUptime: 2807 + 844 ms
installer: app.grapheneos.apps

java.lang.RuntimeException: Error receiving broadcast Intent { act=com.android.vending.archiver.INSTALL_PACKAGE_ARCHIVED.com.arumgames.solitareword flg=0x10 xflg=0x4 pkg=com.android.vending (has extras) } in qef@6d9c578
	at android.app.LoadedApk$ReceiverDispatcher$Args.lambda$getRunnable$0(LoadedApk.java:2063)
	at android.app.LoadedApk$ReceiverDispatcher$Args.$r8$lambda$mcNAAl1SQ4MyJPyDg8TJ2x2h0Rk(LoadedApk.java:0)
	at android.app.LoadedApk$ReceiverDispatcher$Args$$ExternalSyntheticLambda0.run(D8$$SyntheticClass:0)
	at android.os.Handler.handleCallback(Handler.java:1095)
	at android.os.Handler.dispatchMessageImpl(Handler.java:135)
	at android.os.Handler.dispatchMessage(Handler.java:125)
	at android.os.Looper.loopOnce(Looper.java:296)
	at android.os.Looper.loop(Looper.java:397)
	at android.app.ActivityThread.main(ActivityThread.java:9568)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:575)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:847)
Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.Class java.lang.Object.getClass()' on a null object reference
	at j$.util.Optional.<init>(r8-map-id-4c9b873debdfb2f8920080e4f74cdd4bdcce49b6846db08bbe8e3090d4829b36:4)
	at j$.util.Optional.of(r8-map-id-4c9b873debdfb2f8920080e4f74cdd4bdcce49b6846db08bbe8e3090d4829b36:3)
	at qef.onReceive(PG:107)
	at android.app.LoadedApk$ReceiverDispatcher$Args.lambda$getRunnable$0(LoadedApk.java:2055)
	... 11 more
```

--- a/core/java/android/content/pm/PackageInstaller.java
+++ b/core/java/android/content/pm/PackageInstaller.java
@@ -1433,6 +1433,7 @@ public class PackageInstaller {
             fillIn.putExtra(PackageInstaller.EXTRA_PACKAGE_NAME, archivedPackageInfo.getPackageName());
             // there's no unprivileged alternative to this API
             fillIn.putExtra(PackageInstaller.EXTRA_STATUS, PackageInstaller.STATUS_FAILURE);
+            fillIn.putExtra(PackageInstaller.EXTRA_STATUS_MESSAGE, "installPackageArchived() is not allowed");
             com.android.internal.os.BackgroundThread.getHandler().post(() -> {
                 try {
                     statusReceiver.sendIntent(GmsCompat.appContext(), 0, fillIn, null, null);

